cms/static/js/models/settings/course_details.js

```diff
@@ -15,7 +15,8 @@ var CourseDetails = Backbone.Model.extend({
         intro_video: null,
         effort: null,	// an int or null,
         course_image_name: '', // the filename
-        course_image_asset_path: '' // the full URL (/c4x/org/course/num/asset/filename)
+        course_image_asset_path: '', // the full URL (/c4x/org/course/num/asset/filename)
+        pre_requisite_course: ''
```

Comment:
As per @cahrens earlier suggestion to get things ready for multiple pre-requisites, perhaps this should be an array and - for now - assert that len(array) is 1 on the handlers